modules/tinymce/src/core/main/ts/annotate/AnnotationContext.ts

```diff
@@ -36,9 +36,9 @@ const context = (editor: Editor, elem: any, wrapName: string, nodeName: string):
       return ChildContext.Valid;
     } else if (isAnnotation(elem)) {
       return ChildContext.Existing;
-    } else if (isCaretNode(elem)) {
+    } else if (isCaretNode(elem.dom())) {
```

Comment:
This was a bug I found, as this was expecting a dom node, but was being passed a sugar element. This meant the `NodeType.isElement` check would always be false previously.